contributor/making-release.md

```diff
@@ -6,7 +6,43 @@ redirect_from:
   - /governance/releasing
 ---
 
-This is a guide for the creation of a release of Apache Accumulo. 
+Follow these steps to make a release of Apache Accumulo. 
+
+1. [Setup]
```

Comment:
I wonder if this would work `[Setup](#setup)` and then would not need to define links below.